themes/theme-default/src/client/styles/code.scss

```diff
@@ -1,8 +1,8 @@
 @import '_variables';
 
 // Safari only support integer pixel value for line-height
-// the original font-size is 16px, 1.375 will make line-height be 22px
-$line-height: 1.375;
+// the original font-size is 14px, 1.7 will make line-height be 23.8px
```

Comment:
Please double confirm what you are doing here becausecurrently lower versions of safari still on support  integral pixels with line height in pre element

Reply:
The current default theme's line height is based on an incorrect style. Although it appears to perform well, the font sizes of line numbers, highlighted lines, and line code are not actually reset to the same value. The value of 1.375 on line code is actually 0.85em * 1.375 = 18.7, and its good appearance is due to the behavior of the outer `<code>` tag. Once `<code>` is changed to `display: block`, the issues will be exposed. Therefore, modifying all three to 0.875em * 1.7 is the expected solution. The difference between 1.714 and 1.7 in the final rendering performance is not significant for browsers.